entrypoint.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+python manage.py migrate
+python manage.py createsuperuser --noinput
```

Comment:
Usually superusers are created to the database, and not in docker start. Also, it's one time thing, and something that happens differently depending on database purpose and location (local, test, prod).

Reply:
I thought it might be nice to be able to automatically create an initial superuser from environment variables that are set in secrets, during development. If it seems like an unnecessary feature or a risk, I don't mind if it's removed.

Reply:
since the user creds go into database that needs to be handled persistently, I think its place is not in container
usually automated local dev superuser creation would be set when adding fixtures/test data to the db

Reply:
I agree, manually creating `manage.py createsuperuser` or loading it from a fixture is a better idea in many/all cases.